src/main/java/seedu/address/model/person/Person.java

```diff
@@ -67,6 +68,15 @@ public Set<Role> getRoles() {
         return Collections.unmodifiableSet(roles);
     }
 
+    /**
+     * Check is Person is a member
+     * @return true is Person has a role "member"
```

Comment:
LGTM, may want to change "member" to "Member" to keep same with other part in the code